### ISSUE TITLE
Dataset that includes more than 1 distinct population, it loads them together.

### ISSUE THREAD

**Issue by neuroBazzi** [MetaCell/salk-interactive-atlas#186]
From the client

**Comment by afonsobspinto** [comment]
What is a dataset that has more than 1 distinct population?
The rule is:
1 pair data - key files represents 1 population, 

it doesn't matter if the data actually represents multiple populations, if it's coming from the same file it is 1 population for us.

If this is not the intended behavior, most certainly cordmap needs to change.

**Comment by neuroBazzi** [comment]
that rule i clear to them, I think he loaded two files. with 'dataset' he means that that experiment had more than one population. I did ask him to confirm me that he in fact loaded two flies and to share those with me, so stay tuned.

**Comment by neuroBazzi** [comment]
@afonsobspinto they're pushing back on this and would like to change the pipeline to have more than one population in the same file so we will talk about changing cordmap. is there any constraint from your end we should be mindful about when we have this discussion?

**Comment by afonsobspinto** [comment]
Reopening it as a new feature request instead of bug

**Comment by afonsobspinto** [comment]
Cordmap output should have a new file:  cells_with_labels.csv
That file will be equivalent to transformed_cells but with an additional column saying to which population the cell belongs to.

Population label comes from the raw data. Which raises the question:
![image](https://user-images.githubusercontent.com/19196034/187657916-2a21c4b2-9e98-444a-a31f-655b23a6289e.png)
cc @neuroBazzi

**Comment by neuroBazzi** [comment]
@afonsobspinto the name field in the modal is about the name of the experiment, not the name of the population. the two could be the same, but it would be confusing and it is up to the user to not call the experiment the same way as the populations it contains. the experiment name can come, for example, from the subject (i.e. the mouse) name (which is not a name like 'Micky' but more like a code/id). does this help?

**Comment by afonsobspinto** [comment]
You are correct, I got confused for a second. There's no problem regarding the names. Thanks for the clarification
